src/lib.rs

```diff
@@ -227,7 +227,7 @@
 //! }
 //!```
 //! ## Feature Flags
-#![doc = document_features::document_features!()]
+#![cfg_attr(feature = "document-features", doc = document_features::document_features!())]
```

Comment:
To make the section disappear when not enabled rather than having a heading which has no content, also slap this on the header:
```rus
#[cfg_attr(feature = "document-features", doc = "## Feature Flags")]
```

Reply:
Smart! I noticed this and wanted to get rid of it but it didn't occur to me to do this.